lib/src/screens/onboarding_screen.dart

```diff
@@ -0,0 +1,254 @@
+                child: Container(
+                  width: widthScreen,
+                  height: heightScreen,
+                  child: Column(
+                    children: [
+                      SizedBox(height: iconAndTopSpacing),
+                      Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          Image(
+                              image: AssetImage(AppIcons.logoIcon),
+                              width: iconWidth,
+                              height: iconHeight),
+                          SizedBox(width: iconAndTextSpacing),
+                          Text(AppConstants.textIcon,
+                              style: Theme.of(context)
+                                  .textTheme
+                                  .headline6
+                                  ?.copyWith(
+                                    fontFamily: AppFonts.fontAppSemiBold,
+                                    letterSpacing: 0.4,
+                                  ))
+                        ],
+                      )
+                    ],
+                  ),
+                ),
+              ))
+            else if (Device.get().isTablet)
+              (Column(
+                crossAxisAlignment: CrossAxisAlignment.center,
+                children: [
+                  SizedBox(height: iconAndTopSpacing),
+                  Padding(
+                    padding: EdgeInsets.symmetric(horizontal: iconPadding),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Image(
+                          height: iconHeight,
+                          width: iconWidth,
+                          image: AssetImage(AppIcons.logoIcon),
+                        ),
+                        Spacer(),
+                        Text(AppConstants.textIcon,
+                            style: Theme.of(context)
+                                .textTheme
+                                .headline6
+                                ?.copyWith(
+                                    fontFamily: AppFonts.fontAppBold,
+                                    letterSpacing: 0.4,
+                                    height: 1,
+                                    color: Color(
+                                        AppOnBoardingScreen.iconTextColor))),
+                      ],
+                    ),
+                  ),
+                  SizedBox(height: iconAndBottomSpacing),
+                  Padding(
+                    padding: EdgeInsets.symmetric(
+                        horizontal: tutorialShareRecipesTextPadding),
+                    child: Text(AppConstants.tutorialShareRecipesText,
+                        maxLines: 2,
+                        style: Theme.of(context).textTheme.headline3?.copyWith(
+                            letterSpacing: -0.5,
```

Comment:
why we have a negative sign here?

Reply:
![image](https://user-images.githubusercontent.com/46520444/128049097-be9d94ff-78ae-400b-bafe-244a189c7879.png)
Because the letterspacing in design show -0.5. I have coded according this design.